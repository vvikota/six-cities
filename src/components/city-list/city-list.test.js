import React from 'react';
import renderer from 'react-test-renderer';
import CityList from './city-list.jsx';

it(`CityList is correctly renderer`, ()=> {
  const tree = renderer.create(<CityList
    cityList={[]}
    currentCity={`London`}
    onChangeCity={jest.fn()}
    hoverItem={jest.fn()}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
