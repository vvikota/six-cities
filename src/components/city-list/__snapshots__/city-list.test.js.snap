// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityList is correctly renderer 1`] = `
<div
  className="cities tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    />
  </section>
</div>
`;
